pages/import-nft.tsx

```diff
@@ -228,7 +228,7 @@ import { Workflow } from "types/workflow";
      ).then(receipt => {
```

Comment:
Can pull the success callback lines out into a separate function (e.g. `itemListedHandler`) that both `listNewItem` and `listNewAuctionItem` call to reduce the need for updating handler code in two spots.

Both functions would also benefit by being refactored into full use of async/await code structuring with try/catch blocks.

Reply:
Great. I'm on it will push the change

Reply:
Will resolve both locally then commit and push the changes shortly